tutorial/20_recipes/002_multi_objective.py

```diff
@@ -117,6 +117,9 @@ def objective(trial):
 
 print("Number of finished trials: ", len(study.trials))
 
+###################################################################################################
+# Pareto-front as the list of FrozenTrial
+print(study.best_trials)
```

Comment:
It might be difficult for readers to parse the corresponding output cell of this command.
Not a strong opinion, but updating the multi-objective tutorial seems to be out-of-scope of this PR. How about removing this change to keep the PR compact?

![image](https://user-images.githubusercontent.com/3255979/159836598-77a54909-ca6d-4387-940a-7f344a0b1346.png)

Reply:
Okay, got it.